server.go

```diff
@@ -32,6 +33,7 @@ type server struct {
 	authService       auth.Service
 	backgroundService background.Service
 	languageService   language.Service // NOTE: Could put directly service typed as NaturalAPIService, remove Init() from interface and just put it in NaturalAPIService
+	accountService    *communication.AccountsServiceClient
```

Comment:
Why creating a service for connection and don't you just store a `accountsv1.AccountsAPIClient` in the server ?

Reply:
in my opinion it's cleaner to wrap the connection with other services in another package